<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimulationTask3Widget</class>
 <widget class="QWidget" name="SimulationTask3Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>819</width>
    <height>567</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Heart Transform</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QDockWidget" name="dockWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">QToolBox::tab {
         background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                     stop: 0 #E1E1E1, stop: 0.4 #DDDDDD,
                                     stop: 0.5 #D8D8D8, stop: 1.0 #D3D3D3);
         border-radius: 5px;
         color: black;
     }

     QToolBox::tab:selected { /* italicize selected tabs */
         font: bold;
         color: black;
     }
QToolBox {
    padding : 0
}</string>
     </property>
     <property name="floating">
      <bool>false</bool>
     </property>
     <property name="features">
      <set>QDockWidget::NoDockWidgetFeatures</set>
     </property>
     <property name="windowTitle">
      <string>Simulation Task 3</string>
     </property>
     <widget class="QWidget" name="dockWidgetContents">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="widget_2" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>h:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="stepSizeEdit">
            <property name="inputMethodHints">
             <set>Qt::ImhFormattedNumbersOnly</set>
            </property>
            <property name="text">
             <string>0.0</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item alignment="Qt::AlignLeft">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>tolerance:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="toleranceEdit">
              <property name="inputMethodHints">
               <set>Qt::ImhFormattedNumbersOnly</set>
              </property>
              <property name="text">
               <string>1.0e-7</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="simulateButton">
         <property name="text">
          <string>Simulate</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="clearButton">
         <property name="text">
          <string>Clear graph</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="doneButton">
         <property name="text">
          <string>Done</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="plotPane" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>3</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
